setup.py

```diff
@@ -83,7 +83,8 @@ def build_cmake(self, ext):
             '-DPY_VERSION=' + '{v[0]}.{v[1]}'.format(v=version_info),
             '-DBUILD_PYTHON=On',
             '-DBUILD_TESTS=Off',
-            '-DWARNINGS=Off'
+            '-DWARNINGS=Off',
+            '-DBoost_NO_BOOST_CMAKE=ON'
```

Comment:
So this option is okay to be universally applied?

Reply:
It looks like it is. I tested against older versions of boost on linux and windows and no problems came up.

I could pipe a flag through setup,py to disable this, but that would require some plumbing. The pip command to use setup.py options is also pretty nasty, but it might be worth it?

Reply:
Is it the case that older versions just ignore the option? So it is either ignored or needed? In that case it is likely okay?

Reply:
This option has been in cmake since at least 3.0.2, ubuntu 18.04 is currently at 3.10.2.

It seems like the flag only affects boost versions built with cmake (the boost-cmake project). I'm not sure which package libraries use that method, so I can't test against them. So far its worked with OSX/homebrew, Ubuntu/anaconda, WIndows/vcpkg

Reply:
I'll add a flag in setup.py to let us disable this in case it causes problems for some people. I'll also see if I can extend it to add arbitrary cmake flags so we won't need to make a release for every bug we find